### ISSUE TITLE
How to process Raw NBconvert cells using nbpublish -pdf into latex?

### ISSUE THREAD

**Issue by bdatko** [chrisjsewell/ipypublish#118]
I am trying to add raw latex into the tex document when converting with nbpublish -pdf, but I cannot figure out what metadata I need to add, either in the document or cell level, to get nbpublish to recognize the latex when `"cell_type" : "raw"`. It seems nbpublish ignores the raw cells, as designed in the ['opt in' philosophy](https://github.com/chrisjsewell/ipypublish/blob/master/docs/source/nb_conversion.rst#user-content-built-in-export-configurations). Which I agree with, just want raw latex cells to opt in when needed. =)

I can inject raw latex into the tex document when the cell type is markdown, but having the raw cell_type with the a metadata tag would be good .

**Comment by chrisjsewell** [comment]
Yep sounds reasonable. I can't promise that I will be able to get round to doing this in the near future 😐 But I'd welcome any Pull Requests 😀

**Comment by bdatko** [comment]
No problem @chrisjsewell. I thought about my question, and realize it might a redundant enhancement. Seeing how you envisioned the workflow writing Latex in the markdown cell vs the raw cell is a small difference. 

I might take you up on the PR.
